contracts/cw3-dao/src/tests.rs

```diff
@@ -2032,7 +2026,7 @@ fn test_update_staking_contract() {
 
     // Nobody can call call update staking contract method directly
     let update_staking_contract_msg = ExecuteMsg::UpdateStakingContract {
-        new_staking_contract: Addr::unchecked("Better_Staking_Contract"),
+        new_staking_contract: "Better_Staking_Contract".to_string(),
```

Comment:
Heads up: upper case addresses become invalid in the latest MockApi implementation. But you can use "better_staking_contract" when you run into trouble.